### ISSUE TITLE
Wrong message on /offer/create

### ISSUE THREAD

**Issue by magul** [CodeForPoznan/volontulo#714]
__Description:__

When user doesn't have organization and click link in footer `Dodaj nową ofertę` he/she get an information in message, that:
```
Nie masz jeszcze żadnej założonej organizacji na volontuloapp.org. Aby założyć organizację, <a href='/o/organizations/create'>kliknij tu.</a>
```
__Current behaviour:__
Message looks like:
```
Nie masz jeszcze żadnej założonej organizacji na volontuloapp.org. Aby założyć organizację, <a href='/o/organizations/create'>kliknij tu.</a>
```
without a link.

__Expected behaviour:__
Message looks like:
```
Nie masz jeszcze żadnej założonej organizacji na volontuloapp.org. Aby założyć organizację, kliknij tu.
```
and `kliknij tu` is a proper link

**Comment by Rafalkufel** [comment]
Biorę to!

**Comment by Rafalkufel** [comment]
Solved!

**Comment by magul** [comment]
closed in #718
